pyvocz/templates/_base.html

```diff
@@ -42,7 +42,7 @@
       {% if g.lang_code == 'cs' %}
       <div class="notice">
           <p>
-              Vzhledem k aktuální situaci a mimořádným opatřením vydaným v souvislosti s onemocněním COVID-19 rušíme až do odvolnání pravidelné brněnské Pyvo.
+              Vzhledem k aktuální situaci a mimořádným opatřením vydaným v souvislosti s onemocněním COVID-19 rušíme až do odvolání pravidelné brněnské Pyvo.
```

Comment:
Nechceš tam napsat spíš "Pyva v podobě osobních setkání"? Nejde jen o brněnské Pyvo.

Reply:
Nie som doma. Uprav to ako uznas za vhodne.

O ostatnych miestach neviem. Niektore mesta maju eventy a nevyjadrili sa.